css/css-values/calc-angle-values.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Values and Units Test: calc() function with angle values</title>
+
+  <!--
+
+  Original test is:
+
+  https://chromium.googlesource.com/chromium/src/+/c825d655f6aaf73484f9d56e9012793f5b9668cc/third_party/WebKit/LayoutTests/css3/calc/calc-with-time-angle-and-frequency-values.html
+
+  Issue 917718: [css-values] calc-with-time-angle-and-frequency-values
+  test is highly unreliable, transition-delay testing causes side effects
+  https://bugs.chromium.org/p/chromium/issues/detail?id=917718
+
+  -->
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-values-3/#calc-computed-value">
+  <link rel="help" href="https://www.w3.org/TR/css-values-3/#angles">
+
+  <meta content="This test checks that additions, substractions, multiplications and divisions in calc() function when applied to angle units." name="assert">
+
+  <script src="/resources/testharness.js"></script>
+
+  <script src="/resources/testharnessreport.js"></script>
+
+  <div id="log"></div>
+
+  <div id="target"></div>
+
+  <script>
+  function startTesting()
+  {
+
+    var targetElement = document.getElementById("target");
+
+    function compareValue(property_name, calcValue, expectedValue, description)
+    {
+
+    test(function()
+      {
+
+      targetElement.style.setProperty(property_name, "inherit");
+
+      /*
+      Since we are running many consecutive tests on the same
+      element, then it is necessary to 'reset' its property
+      to an initial value before actually re-testing it.
+      */
+
+      targetElement.style.setProperty(property_name, calcValue);
+
+      var computedCalcValue = getComputedStyle(targetElement)[property_name];
+
+      targetElement.style[property_name] = expectedValue;
+
+      var computedExpectedValue = getComputedStyle(targetElement)[property_name];
+
+      assert_equals(computedCalcValue, computedExpectedValue, calcValue + ' should equal to ' + expectedValue);
+
+      targetElement.style.removeProperty(property_name);
+      }, description);
+    }
+
+    /* Testing angle units  */
+
+    /*
+
+    deg
+       Degrees. There are 360 degrees in a full circle.
+
+    grad
+       Gradians, also known as "gons" or "grades". There are 400 gradians in a full circle.
+
+    rad
+       Radians. There are 2π radians in a full circle.
+
+    turn
+       Turns. There is 1 turn in a full circle.
+
+   */
+
+
+ /* compareValue(property_name, calcValue, expectedValue, description)  */
+
+    compareValue("transform", "rotate(calc(45deg + 45deg))", "rotate(90deg)", "addition of 2 angle units: deg plus deg");
+
+    compareValue("transform", "rotate(calc(45deg + 1rad))", "rotate(102.3deg)", "addition of 2 angle units: deg plus rad");
```

Comment:
Please use something like shouldBeCloseTo()

Reply:
Inserting Math.PI into the functions works as expected. Eg

http://www.gtalbot.org/BrowserBugsSection/CSS3Values/calc-Math.PI-rad-and-grad.html

I will adapt+adjust calc-angle-values.html test to use Math.PI

Reply:
I have create compareValuesCloseTo which defines a tolerance precision (epsilon):

http://www.gtalbot.org/BrowserBugsSection/CSS3Values/calc-angle-values-latest.html

I am closing this Pull Request since I have submitted a new one 
https://github.com/web-platform-tests/wpt/pull/15342
with all of the requested improvements in here.